src/test/resources/org/broadinstitute/hellbender/tools/walkers/qc/pileup/expectedPileup.txt

```diff
@@ -0,0 +1,238 @@
+1 200 N A B [Feature(s): [VC Unknown @ 1:199-200 Q232.46 of type=INDEL alleles=[GG*, G] attr={} GT=[], [VC Unknown @ 1:200 Q232.46 of type=SNP alleles=[G*, A] attr={} GT=[]] -130
```

Comment:
hmm, every single reference base here seems to be `N` - can you pick inputs so that actual reference bases (non-N ones) would show up here?